src/components/core/core-class.js

```diff
@@ -167,7 +167,7 @@ class Swiper extends SwiperClass {
       touchEvents: (function touchEvents() {
         const touch = ['touchstart', 'touchmove', 'touchend'];
         let desktop = ['mousedown', 'mousemove', 'mouseup'];
-        if (window.navigator.pointerEnabled) {
+        if (window.navigator.pointerEnabled || window.PointerEvent) {
```

Comment:
I notice multiple checks of the same properties. Could this be distilled into a single property to check?
This would make changes in the future easier too.

Reply:
What about older implementions and old browsers?

Reply:
It's not removing support for older browsers it's moving all of the duplicated checks to a property.
So instead of `if (a || b || c || d || e)` in multiple places it becomes `if (check.foo)` in multiple places.

Reply:
If you mean my change: I just added it where it is needed.
I agree that we should moe it to one global check.

Reply:
@nolimits4web what do you think?

Reply:
> I agree that we should moe it to one global check.

Ya, maybe better for a follow-up PR separate from this.

Reply:
Right, might be candidate for some refactoring in a separate PR.